### DISCUSSION TITLE
Proyecto entrega 1

### DISCUSSION THREAD

**Discussion by lguerra98** [rramosp/ai4eng.v1#150] [UDEA Students]
Hola buenas noches, queria saber como les parece los datos de esta competencia kaggle que quiero utilizar para el proyecto, creo que no cumple especificamente con los parametros de las columnas que se piden pero el resto de la base de datos me parece tine buen contenido para realizar el proyecto,

link competencia: https://www.kaggle.com/competitions/store-sales-time-series-forecasting/dat

gracias.

**Comment by rramosp** [comment]
hola lguerra, creo que el dataset está bien. tiene la información
distribuida en varias tablas con lo que las tendrás que integrar. eso
compensa que haya un poco menos de columnas

On Wed, Jun 29, 2022 at 1:13 AM lguerra98 ***@***.***> wrote:

> Hola buenas noches, queria saber como les parece los datos de esta
> competencia kaggle que quiero utilizar para el proyecto, creo que no cumple
> especificamente con los parametros de las columnas que se piden pero el
> resto de la base de datos me parece tine buen contenido para realizar el
> proyecto,
>
> link competencia:
> https://www.kaggle.com/competitions/store-sales-time-series-forecasting/dat
>
> gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/150>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPNCRTBZLTETANLDB3VRPSPHANCNFSM52EP54SA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
